app/reducers/index.js

```diff
@@ -1,9 +1,7 @@
-import {combineReducers} from 'redux';
-import {users, user} from './userReducer';
-
+import { users, user} from './userReducer';
+import { combineReducers } from 'redux';
 const rootReducer = combineReducers({
-  users,
-  user
+  user: user,
```

Comment:
This change was on purpose because escodegen (the library used to generate new code), use this convention ( key: value) although it's not required. So I did that change in order to keep consistency between the existing and the new generated code.